src/main.go

```diff
@@ -606,8 +660,8 @@ func (t *tester) concurrentExecute(querys []query, wg *sync.WaitGroup, errOccure
 	}
 }
 
-func (t *tester) loadQueries() ([]query, error) {
-	data, err := os.ReadFile(t.testFileName())
+func (t *tester) loadQueries(fileName string) ([]query, error) {
```

Comment:
Should we have a new `tester` for a new `--source`'d file instead of using the upper level `tester` with a different filename argument?

Reply:
I don't think so, the included files are still part of the same test.